<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePlayer</class>
 <widget class="QMainWindow" name="SimplePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="LeftDown">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üô</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Down">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üì</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="RightDown">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üò</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Left">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üê</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Home">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>üè†</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Right">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üí</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="LeftUp">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üñ</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Up">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üë</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="RightUp">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>‚Üó</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="zoomOut">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomIn">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Zoom">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="TSRV">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TSRV Operator Control Panel</string>
    </property>
   </widget>
   <widget class="QLabel" name="BEC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>661</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="VlcWidgetVideo" name="video_2" native="true">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="VlcWidgetVideo" name="video_3" native="true">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="VlcWidgetVideo" name="video_4" native="true">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="VlcWidgetVideo" name="video" native="true">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="openUrl">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Live Stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openLocal">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Local File</string>
    </property>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpenLocal">
   <property name="text">
    <string>Open local file</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="text">
    <string>Open URL</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SimplePlayer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
